### DISCUSSION TITLE
My domain registered via vercel is receiving ERR_CERT_AUTHORITY_INVALID

### DISCUSSION THREAD

**Discussion by cjmling** [vercel/community#2135] [Help]
### Summary

Firefox Error code: SEC_ERROR_UNKNOWN_ISSUER
Chrome: NET::ERR_CERT_AUTHORITY_INVALID

Just started happening today.

I am not able to access the website right now? Is there something I can do to fix this or Vercel team have to fix this for me?

**Comment by swarnava** [comment]
Hi, Can you share the domain?
